lib/FocusTrap.js

```diff
@@ -1,31 +1,91 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import ReactDOM from 'react-dom';
+import {addFocusListener, addBlurListener} from './focusListeners';
 
 class FocusTrap extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.focusListener;
```

Comment:
What's the purpose of these two lines? They are not assignments, but do they perform some sort of initialisation I am not aware of?